share/locale/musescore_es.ts

```diff
@@ -19568,7 +19568,7 @@ Traste %1 en cuerdas %2</translation>
         <location filename="../../src/notation/view/widgets/pagesettings.cpp" line="170"/>
         <source>in</source>
         <comment>abbreviation of inch</comment>
-        <translation type="unfinished"/>
+        <translation>en</translation>
```

Comment:
I don't speak Spanish, but somehow I doubt whether this translator saw the message "abbreviation of inch"